libs/common-services/src/main/java/uk/gov/di/ipv/core/library/config/ConfigurationVariable.java

```diff
@@ -8,7 +8,7 @@ public enum ConfigurationVariable {
     CIMIT_CONFIG("cimit/config"),
     CIMIT_SIGNING_KEY("cimit/signingKey"),
     CIMIT_API_BASE_URL("cimit/apiBaseUrl"),
-    CIMIT_API_KEY("cimitApi/api-key"),
+    CIMIT_API_KEY("cimitApi/apiKey"),
```

Comment:
I'd leave this as `api-key` for now - I have a core-back PR that will introduce all these changes in one go